qiskit_experiments/library/tomography/fitters/lstsq_utils.py

```diff
@@ -212,55 +228,62 @@ def dirichlet_mean_and_var(
         outcome_shots = np.sum(outcome_data, axis=(0, -1))
         posterior_shots = np.sum(posterior, axis=(0, -1))
         posterior_total = posterior_shots + shot_data - outcome_shots
+    posterior_total = posterior_total[None, 0, None]
+
+    # Posterior mean and variance
+    mean_probs = posterior / posterior_total
+    variance = mean_probs * (1 - mean_probs) / (posterior_total + 1)
 
-    # Fill matrices
-    for i in range(dim_circ_cond):
-        cond_idxs = {i: 0 for i in range(dim_cond)}
-        for j in range(num_data):
-            freqs = posterior[i, j]
-            for outcome in range(tomo_outcomes):
-                shots = posterior_total[outcome]
-                outcome_cond = f_cond_outcome(outcome)
-                idx = cond_idxs[outcome_cond]
-                mean_probs[i, outcome_cond, idx] = freqs[outcome] / shots
-                var_denom[i, outcome_cond, idx] = shots + 1
-                cond_idxs[outcome_cond] += 1
-    variance = mean_probs * (1 - mean_probs) / var_denom
     return mean_probs, variance
 
 
+# pylint: disable = bad-docstring-quotes
+@deprecate_function(
```

Comment:
Missing in release note.

Reply:
This is an internal function not in API docs so i didnt think it should go in release note, but i added this warning in case someone had dug it out from source code

Reply:
fair enough